src/App.jsx

```diff
@@ -1,29 +1,39 @@
-import React, { Component } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import './App.css';
-import Header from './Header';
-import Questions from './Questions';
-import Answer from './Answer';
-import QuestionData from './Questions.json';
-
-class App extends Component {
-  render() {
-    return (
-      <Router>
-        <div className="Content">
-          <Header />
-          <h2>Manager is ready to help you!</h2>
-          <Route exact path="/" component={Questions} />
-          <Route
-            path="/question/:questionId"
-            render={({ match }) => (<Answer data={QuestionData.find(item =>
-            item.id.toString() === match.params.questionId)}
-            />
-       )} />
-        </div>
-      </Router>
-    );
-  }
-}
-
-export default App;
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import './App.css';
+import Header from './Header';
+import Questions from './Questions';
+import Answer from './Answer';
+import QuestionData from './Questions.json';
+
+class App extends Component {
```

Comment:
I think this component should be renamed from App.jsx to Router.jsx because there's only routing logic
and the name App doesn't fit the component's semantic. Also component's name should let other developers intuitively understand what the component is all about without looking inside.